runtimes/common/src/dot_names/mod.rs

```diff
@@ -58,7 +58,10 @@ impl<const MIN_LENGTH: u32, const MAX_LENGTH: u32> TryFrom<Vec<u8>> for DotName<
 	type Error = DotNameValidationError;
 
 	fn try_from(value: Vec<u8>) -> Result<Self, Self::Error> {
-		ensure!(value.len() >= MIN_LENGTH.saturated_into(), Self::Error::TooShort);
+		ensure!(
```

Comment:
Why are these changes here? Do you use a different fmt setup? Are you using the right nightly to format these?

Reply:
I think I do. 

These are vscode settings. The tool chain is installed.
```
  "rust-analyzer.rustfmt.extraArgs": [
        "+nightly-2023-10-02"
  ],
```
Not sure why this is happening

Reply:
saved without fmt in: [20432ec](https://github.com/KILTprotocol/kilt-node/pull/818/commits/20432ec66223bcc1977eccee118695f2ce62f12c)

Reply:
you added the `<usize>` right? how come this was suddenly necessary?

Reply:
Oh, yes. I think I executed `cargo clippy --fix`. I try to investigate why some unrelated crates requires some changes